helpers/notification_helper.py

```diff
@@ -40,6 +49,55 @@ def send_subscription_update(cls, user_id, sending_device_key=""):
         notification.send(clients)
 
     @classmethod
+    def send_upcoming(cls, live_events):
+        from helpers.match_helper import MatchHelper
+        now = datetime.datetime.utcnow()
+        for event in live_events:
+            matches = event.matches
+            next_match = MatchHelper.upcomingMatches(matches, num=1)
+            if next_match[0] and not next_match[0].push_sent:
+                match = next_match[0]
+                if match.time is None or (now.day == match.time.day and match.time + datetime.timedelta(minutes=-15) <= now):
```

Comment:
Why are you checking for `now.day == match.time.day` the only events are live events?